website/project/model.py

```diff
@@ -92,29 +93,40 @@ class MetaSchema(StoredObject):
     # Version of the schema to use (e.g. if questions, responses change)
     schema_version = fields.IntegerField()
 
+    @property
+    def requires_approval(self):
```

Comment:
Am I correct in assuming that these methods are prereg-only? If so, a comment stating so would be helpful.

If they are more generally applicable, consider either defining these items directly in every schema, or adding them to `ensure_schema_structure`. 

The fewer places that we perform runtime modifications on the default contents of a prereg question schema, the better for maintainability.